Categories/Power.lua

```diff
@@ -83,9 +130,7 @@ DogTag:AddTag("Unit", "FractionalMP", {
 
 
 DogTag:AddTag("Unit", "Mana", {
-	code = function(unit)
-		return UnitPower(unit, SPELL_POWER_MANA)
-	end,
+	alias = ("MP(unit=unit, type=%q)"):format(L["Mana"]),
```

Comment:
Should be formatted with `MANA:lower()`, since `L["Mana"]` might not be the same as `MANA`.

Likewise for MaxMana.

Reply:
You're right. I originally had it that way, but there's lots of other places in the mod that use `L["Mana"]` so I wanted to stay consistent. Since the array key uses `MANA`, that's definitely the better option.